modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -26,24 +26,24 @@ spec:
      type: kafka <4>
      url: tls://kafka.example.devlab.com:9093/app-topic <5>
      secret:
-       name: kafka-secret <6>
+      name: kafka-secret <6>
```

Comment:
@jcantrill Is this enough, or do we need to go into more detail?

Reply:
for kafka, i believe the only things we support in the secret are the keys related to tls certs